CHANGELOG.md

```diff
@@ -143,84 +143,85 @@ v5.0.0 release.
 
 Semver-major Changes since v5.0.0
 
+* [[`4e46931406`](https://github.com/nodejs/node/commit/4e46931406)] - **(SEMVER-MAJOR)** **src**: deprecate undocumented variables (Jackson Tian) [#1838](https://github.com/nodejs/node/pull/1838)
```

Comment:
it looks like these are order, except for this one. Maybe move it down with the other `src` commits